.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -s settings.xml -P spring
```

Comment:
-1 for changing the name to `.settings.xml`.  If we are going to require it's use, it shouldn't be hidden.  Either set it up so it's not needed, or don't hide it IMHO.